src/components/Contentful/_contentful_new_design.scss

```diff
@@ -44,6 +44,10 @@
 
     &.shaded {
       background-color: $white-two;
+
+      @include mobile {
+        padding: 50px 0;
```

Comment:
I think it'd be a good idea to introduce another variable for grid-padding on mobile as is used in multiple places